docs/DeveloperGuide.md

```diff
@@ -172,8 +174,25 @@ Below is the Sequence Diagram in completing a quiz.
 Fig #. Interactions of Components for a Quiz submission
 
 #### 3.2.2 Create Quiz (Educator only)
-In addition to tailored content, Educators can also create their own `Question` for their own Class
+In addition to tailored content, Educators can also create their own `Question` for their own Class. The ER Diagram below zooms in on the relationships used in the creation of quizzes by an Educator.
+
+![Educator Quiz](diagrams/QuizEntity.png)<br>
+Fig #. Relationships of Educators and Quizzes
+
+In creating a Quiz, and its questions, the interactions can be shown in the following Sequence Diagram.
+
+{Create Quiz Sequence} <br>
+Fig #. Interactions of Components for a Quiz Creation
+
+When attempting a quiz created by an educator, it is different from the general quiz mentioned in the earlier Section 3.2.1. 
+In this case, there is no adaptive testing, so the questions are given as they were added by the educator.
+The User Guide (Educator) demonstrates the creation of such a quiz, and the User Guide (Student) showcases attempting the quiz.
+To the Student, both quizzes are similar in front-end design. The only difference is the back-end implementation of retrieving the next question.
+
+
+
 ### 3.3 Adaptive Testing <a name="adaptive"></a>
+Separate to quizzes created by an Educator, the platform also s
```

Comment:
Is there a typo or incomplete sentence here?

Reply:
Looks like incomplete.